evaluation/webarena/scripts/run_infer.sh

```diff
@@ -35,7 +40,7 @@ COMMAND="poetry run python evaluation/webarena/run_infer.py \
   --llm-config $MODEL_CONFIG \
   --max-iterations 15 \
   --max-chars 10000000 \
-  --eval-note $EVAL_NOTE"
```

Comment:
Are the removals of EVAL_NOTE intended?

Reply:
No, it was a mistake. Thanks for the catch!